tests/parser/exceptions/test_constancy_exception.py

```diff
@@ -17,18 +17,21 @@
 @constant
 def foo() -> int128:
     self.x = 5
+    return 1
     """,
     """
 @public
 @constant
 def foo() -> int128:
     send(0x1234567890123456789012345678901234567890, 5)
+    return 1
     """,
     """
 @public
 @constant
 def foo() -> int128:
     selfdestruct(0x1234567890123456789012345678901234567890)
```

Comment:
I feel like this one should fail to compile, since a pathway (the only one) can `selfdestruct` and fail silently to the caller

Reply:
That's an interesting observation, because SELFDESTRUCT is defined by the yellow paper to be a normal halting condition. That means the only way the caller can distinguish from a return with data is by the length of the return data buffer. And it cannot be distinguished at all from a return without data.

But I think that is out of scope of this PR, rather a rule should be added to the type checker to treat `selfdestruct(...)` the same as `return <no arguments>`.

Reply:
Yup, I agree. And two returns in series should be a compiler error

Reply:
Added checker for multiple exits on a single 'body' level.